app/webroot/index.php

```diff
@@ -44,6 +44,13 @@
 }
 
 /**
+ * Config Directory
+ */
+if (!defined('CONFIG')) {
+	define('CONFIG', ROOT . DS . APP_DIR . DS . 'Config');
```

Comment:
What about all the apps out there that don't have this constant defined? Won't they break when the upgrade?

Seems like this fallback should be part of the CakePHP bootstrap process.